### DISCUSSION TITLE
NetworkError

### DISCUSSION THREAD

**Discussion by ShuoShenDe** [foxglove/community#585] [Q&A]
**Description**
 ![Screenshot from 2022-12-22 20-38-09](https://user-images.githubusercontent.com/31400000/209213356-15f724e8-0025-4d62-acdf-d7143165bae5.png)
My network is good for internet, but the studio gives me an error of NetworkError. The web server has not error logs.
![Screenshot from 2022-12-22 20-42-03](https://user-images.githubusercontent.com/31400000/209213711-c16d7c7f-c4d9-4153-adc6-9f15bb43fe94.png)

- Version:
node -v
v16.18.1
npm -v
8.19.2

- Platform:
VSCode
Ubuntu 18

**Steps To Reproduce**

Not sure

**Expected Behavior**
How to find out the problem?

**Comment by jtbandes** [comment]
What is the network request that failed?

**Comment by jtbandes** [comment]
Moving to a discussion since this seems to be an issue with the development of studio rather than with the production app.
